frontend/frontend.py

```diff
@@ -293,7 +293,122 @@ def img2img_submit_params():
 
                 img2img_width.change(fn=uifn.update_dimensions_info, inputs=[img2img_width, img2img_height], outputs=img2img_dimensions_info_text_box)
                 img2img_height.change(fn=uifn.update_dimensions_info, inputs=[img2img_width, img2img_height], outputs=img2img_dimensions_info_text_box)
-
+            
+            with gr.TabItem("Image Lab", id='imgproc_tab'):
+                    gr.Markdown("Post-process results")
+                    with gr.Row():
+                        with gr.Column():
+                            with gr.Tabs():
+                                with gr.TabItem('Single Image'):
+                                    imgproc_source = gr.Image(label="Source", source="upload", interactive=True, type="pil")
+
+                            #gfpgan_strength = gr.Slider(minimum=0.0, maximum=1.0, step=0.001, label="Effect strength",
+                            #                            value=gfpgan_defaults['strength'])
+                            #select folder with images to process
+                                with gr.TabItem('Batch Process'):
+                                    imgproc_folder = gr.File(label="Batch Process", file_count="multiple",source="upload", interactive=True, type="file")
+                            with gr.Row():
+                                imgproc_btn = gr.Button("Process", variant="primary")
+                            gr.HTML("""
```

Comment:
Minor: this (and the other text sections) would be a lot easier to maintain as markdown instead of html markup. Centering the text also makes it harder to read.

Reply:
Well first of all I agree that it should live in the wiki, the problem is that there's a large subset of peeps that are getting into AI generated stuff while having no knowledge about more advanced topics, I can't expect people to know what GoBig is or Latent Diffusion, which is why I included some small info there, plus it pads out the UI to make it look better when the upscale options are open lol.

In regards to HTML vs Markdown, I actually had it as Markdown before, but decided to use HTML instead so I can use the same style as the message at the bottom of the UI, may be it should be defined outside of the loop tho.

Reply:
The message in the footer is long. This one is multiple paragraphs, and it's nested inside of a column in a container. It would really be easier to read if it was left-aligned normally :sweat_smile: (that space on the left side should be reserved for the controls/settings probably?)

Reply:
Amm hard for me to tell, on my screen it looks fine, but I evidently didn't try it on others, what device are you viewing it on?